### PR TITLE
[Nexus-8433] Adds masks to Capabilities, Tasks, and LDAP

### PR DESCRIPTION
**Pull request by djsauble**
Issues:
https://issues.sonatype.org/browse/NEXUS-8432
https://issues.sonatype.org/browse/NEXUS-8433
https://issues.sonatype.org/browse/NEXUS-8443

Related to: https://github.com/sonatype/nexus-pro/pull/527

Changes:
* Adds masks to Capabilities, Tasks, and LDAP.
* Standardizes on three types of masks:
  * Form masks (prevent changes to a particular SettingsForm)
  * Feature masks (prevent changes to the current feature)
  * Full-page masks (prevent changes to the entire application), may include a modal dialog

**I have deliberately ignored legacy parts of the app, such as Upload, Staging, and Browse Repositories, as those will be rewritten for NX3.**

### CODE REVIEW

**Comment by jdillon** [comment]
@djsauble capabilities mask on each section seems uneeded and costs a bit of extra code, vs. masking the feature view.

**Comment by djsauble** [comment]
@jdillon Agreed. As discussed yesterday, we need three different types of masks:

* Form masks (prevent changes to a particular SettingsForm)
* Feature masks (prevent changes to the current feature)
* Full-page masks (prevent changes to the entire application), may include a modal dialog

I plan to implement these as part of https://issues.sonatype.org/browse/NEXUS-8432

**Comment by djsauble** [comment]
@jdillon Modal types implemented. https://github.com/sonatype/nexus-oss/commit/ebce1eefed32a0da76cc99654387f7b22747974d

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/LdapServers.js [hunk 1]
```diff
@@ -36,7 +36,6 @@ Ext.define('NX.coreui.controller.LdapServers', {
     'LdapSchemaTemplate'
   ],
   views: [
-    'ldap.LdapServerAdd',
```
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/LdapServers.js]: this just cleanup and unrelated to the masking?

Comment (djsauble) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/LdapServers.js]: Yep. That view was replaced and never removed.

**Pull request closed without merging**